{
  "Header": {
    "Original": "CVE-2018-18385 (Asciidoctor in versions \u0026lt; 1.5.8 allows remote attackers to cause a  ...)",
    "Line": 275141,
    "ID": "CVE-2018-18385",
    "Description": "(Asciidoctor in versions \u0026lt; 1.5.8 allows remote attackers to cause a  ...)"
  },
  "Annotations": [
    {
      "Original": "- asciidoctor 1.5.8-1 (low; bug #913892)",
      "Line": 275142,
      "Type": "package",
      "Package": "asciidoctor",
      "Kind": "fixed",
      "Version": "1.5.8-1",
      "Severity": "low",
      "BugNo": 913892
    },
    {
      "Original": "[stretch] - asciidoctor \u003cno-dsa\u003e (Minor issue)",
      "Line": 275143,
      "Type": "package",
      "Release": "stretch",
      "Package": "asciidoctor",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[jessie] - asciidoctor \u003cno-dsa\u003e (Minor issue)",
      "Line": 275144,
      "Type": "package",
      "Release": "jessie",
      "Package": "asciidoctor",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/asciidoctor/asciidoctor/issues/2888",
      "Line": 275145,
      "Type": "NOTE",
      "Description": "https://github.com/asciidoctor/asciidoctor/issues/2888"
    }
  ]
}